pkg/pillar/types/locationconsts.go

```diff
@@ -51,6 +51,8 @@ const (
 	// ServerSigningCertFileName - filename for server signing leaf certificate
 	ServerSigningCertFileName = CertificateDirname + "/server-signing-cert.pem"
 
+	// ShareCertDirname - directory to place private proxy server certificates
+	ShareCertDirname = "/usr/local/share/ca-certificates"
```

Comment:
Don't we already have the certificates in /etc/ssl/certs and /hostfs/etc/ssl/certs? Do we need a third set of hundreds of kilobytes of certs? (I'm assuming the package you added puts all of the default certs in this directory.)

Reply:
No, this is not another set of certs.
this is to add a new cert into the existing certs, some online url: https://manuals.gfi.com/en/kerio/connect/content/server-configuration/ssl-certificates/adding-trusted-root-certificates-to-the-server-1605.html

here is the detail on the device:
1. on the EVE, when boots up check /etc/ssl/certs dir, 
(ns: pillar) 10d0ffa2-8df1-4c1b-be74-a8b30b0333e1:/etc/ssl/certs# grep "END CERT" ca-certificates.crt | wc -l
150

2) check /usr/share/ca-certificates dir:
(ns: pillar) 10d0ffa2-8df1-4c1b-be74-a8b30b0333e1:/usr/local/share/ca-certificates# ls -lt
total 0

3) put a file there:
(ns: pillar) 10d0ffa2-8df1-4c1b-be74-a8b30b0333e1:/usr/local/share/ca-certificates# ls
proxy.pem

4) run the update-ca-certificate command
(ns: pillar) 10d0ffa2-8df1-4c1b-be74-a8b30b0333e1:/usr/local/share/ca-certificates# /usr/sbin/update-ca-certificates

5) check the number of certs in /etc/ssl/certs file:
(ns: pillar) 10d0ffa2-8df1-4c1b-be74-a8b30b0333e1:/usr/local/share/ca-certificates# grep "END CERT" /etc/ssl/certs/ca-certificates.crt | wc -l
151

which increased by one.

Reply:
and the 150 certs are in /usr/share/ca-certifcates/mozilla directory:
(ns: pillar) 10d0ffa2-8df1-4c1b-be74-a8b30b0333e1:/usr/share/ca-certificates/mozilla# ls -lt ca*.crt | wc -l
150

Reply:
Ah so the additional package is to get the update-ca-certificate command, not to get a list of certs. Makes sense.